components/buttons/OpenInStudioButton.js

```diff
@@ -0,0 +1,16 @@
+import Button from './Button'
+import IconRocket from '../icons/Rocket'
+
+export default function OpenInStudioButton() {
+  const sampleSpec = encodeURI('https://raw.githubusercontent.com/asyncapi/asyncapi/v2.2.0/examples/simple.yml')
+  return (
+    <Button
+      className="block mt-2 md:mt-0 md:inline-block md:ml-2"
+      bgClassName="bg-green-500"
+      text="Open Studio"
```

Comment:
@fmvilas Uhh I think both of your suggestions are wrong 😂😬😬😬. 

That is.. if my understanding is correct, we are enabling our "Studio: Comming SOON!" nav button that will lead to opening the tool. If we are indeed opening a tool, then we are simply "Opening Studio."

I would leave the CTA as `Open Studio`. 


I do not think we are opening *IN* our tool (named Studio) nor are we opening *ON* our tool (named Studio). We are simply opening our tool, named Studio.

![Difference Between In and On](https://user-images.githubusercontent.com/19964402/139484946-58839a66-d029-4725-a13b-b20ec81164ad.png)

Reply:
Yeah, that I get it 😄 

The context I'm talking about is "opening the demo in the left **on/in** Studio".

![Captura de pantalla 2021-11-01 a las 11 38 44](https://user-images.githubusercontent.com/242119/139659236-9e6e2ce1-c940-4edc-a653-469b0808926a.png)

It's true though that this is not the line we should change then.

Reply:
(1) Ooooh, I see, I didn't follow before! 😂 That screenshot helps me place what you mean now. Ok, for this section where we are talking about *opening the demo*, it would be `in`. 😀👍🏽

> "Let's open this demo in Studio!"

(2) Correct, for the Nav <li> item in our menu, it should be simply mentioning how to open the tool itself: 

> "Open Studio"

Reply:
Ok, so I will leave with `in`. Thanks! :)

Reply:
Sorry for the confusion, @magicmatatjahu. In this line, it should be "Open Studio". In the example alongside the code, it should be "Open in Studio". So you should only change it here: https://github.com/asyncapi/website/blob/master/components/Hero.js#L21

Reply:
🤦🏼  I know that it should be changed on animation but forgot that I used that button in two places

Reply:
Changed

![image](https://user-images.githubusercontent.com/20404945/139834259-ac674c61-2482-48e4-bd11-d541e00cbb01.png)